scripts/init.js

```diff
@@ -85,6 +85,8 @@ function getPackageName(platform) {
       return "react-native";
     case "macos":
       return "react-native-macos";
+      case "visionos":
+        return "react-native-macos";
```

Comment:
There's a lot of switch statements like these.. changing the default may be hard.

Reply:
I hope this helps in JS land: https://github.com/microsoft/react-native-test-app/pull/1890